templates/zoo.cfg.j2

```diff
@@ -5,11 +5,7 @@ clientPort={{ client_port }}
 initLimit={{ init_limit }}
 syncLimit={{ sync_limit }}
 
-{#
-Also consider:
-server.{{server.id}}={{server.host}}:2888:3888
-#}
 
 {% for server in zookeeper_hosts %}
-server.{{loop.index}}={{server.host}}:2888:3888
+server.{{server.id}}={{server.host}}:2888:3888
```

Comment:
I have thought of something similar, but maybe it could be even simpler 

```yaml
{% for server_host in zookeeper_hosts %}
  server.{{loop.index}}={{server_host}}:2888:3888
{% endfor %}
```

then passing zookeeper hosts would be as simple as:

```yaml
    - role: ansible-zookeeper
      zookeeper_hosts: "{{ groups['zookeepers'] }}"
```